### DISCUSSION TITLE
Create a openfoodfacts-query for producers data

### DISCUSSION THREAD

**Discussion by alexgarel** [openfoodfacts/openfoodfacts-query#151] [Ideas]
* specific to *.pro.openfoodfacts.org
* it must have the owner-tags as an important data
* we have to setup a redisstream for it 
* product opener must filter on owner-tag for normal users (and let moderators access all)

**Comment by john-gom** [comment]
Note that owners_tags is already included in off-query. e.g. 
`curl -d '[{"$match":{}},{"$group":{"_id":"$owners_tags"}}]' -H "Content-Type: application/json" https://query.openfoodfacts.org/aggregate`

**Comment by alexgarel** [comment]
@john-gom the idea here would be to be able:

1. to use off-query to compute facets for producers data
2. to also have events on this platform.
